src/task_processor/migrations/0013_add_last_picked_at.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 4.2.18 on 2025-04-03 07:34
+
+from django.db import migrations, models
+import django.utils.timezone
+import os
+from common.migrations.helpers import PostgresOnlyRunSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("task_processor", "0012_add_locked_at_and_timeout"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="recurringtask",
+            name="last_picked_at",
+            field=models.DateTimeField(
+                auto_now_add=True, default=django.utils.timezone.now
```

Comment:
Commenting here just because I spent some time looking at this and trying to understand if it was an issue or not. In conclusion, I don't think it is, but I'm noting down my thoughts in case anyone else dwells on it. 

My initial thought was "well, if the time here is set to the same for all the records, then we're just going to be in the same place and this field will never become useful" ... but then I realised that once a task gets picked up, this will be updated and it will be at the end of the queue (I think). 

I guess the question is whether it would make more sense to make this field nullable, and then use e.g. `.asc(nulls_first=True)`?